src/helpers/dateUtil.ts

```diff
@@ -0,0 +1,13 @@
+export const datesAreOnSameDay = (first: Date, second: Date) =>
```

Comment:
Try to implement something like this. It's more pretty to use, no need to call each function separately.
https://www.w3schools.com/jsref/jsref_prototype_date.asp

Reply:
I am not sure that it is a good idea to modify standard class prototype. It also won't make things easier to use, it will still be the function that you need to call with some parameters.

Reply:
1) Why it's not good idea? 
2) Yes, but you don't need to import each function every time

Reply:
1. Because you basically need to know that this modification was made to the standart class. Usually people know what strandart type provides and don't expect any additional functionality. If it was a some custom type that would be another story.
2. I am not very good at this area, but I guess it is shouldn't be a big problem. We ussually import a lot of functions.